workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/utils/TicketPayloadMapper.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {TicketPayload} from '../types';
+
+export function TicketPayloadMapper(ticket: TicketPayload) {
```

Comment:
This is just a function not a component, so the first letter shouldn't be capitalized. Usually we call something like this `normalizeTicket`. The parameter should be called `ticketPayload` since you've indicated that's it type. It would also be helpful to add the return type to the function, since this should return a `Ticket`, right?